src/components/photolist/PhotoListView.tsx

```diff
@@ -140,7 +142,7 @@ function PhotoListViewComponent(props: Props) {
     });
   };
 
-  const handleClick = (event, item) => {
+  const handleClick = (event: { shiftKey: any }, item: { id: any }) => {
     // if an image is selectable, then handle shift click
     if (selectable && event.shiftKey) {
       const lastSelectedElement = selectionStateRef.current.selectedItems.slice(-1)[0];
```

Comment:
could be not use `selectionStateRef.current.selectedItems.at(-1)` to get the last selected item?

Reply:
I'd be more than happy to use modern techniques but according to [this](https://github.com/LibrePhotos/librephotos-frontend/blob/dev/package.json#L32) we support IE11. And according to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#browser_compatibility) `Array.prototype.at` does not work on IE11.

Reply:
oh my.. I didn't see that. @derneuere   do we really support IE11? It's gone out of support, so I don't really see a reason why we should support it..

Reply:
I think it makes sense to drop IE11 support :)

Reply:
done